examples/python/RudderKick.py

```diff
@@ -0,0 +1,133 @@
+# RudderKick.py
+#
+# Simulate a pilot performing a rudder kick test by inputing a rudder input based
+# on a ramp input. Aileron input is also included to maintain a steady heading 
+# sideslip (SHSS). The time histories of the control inputs and beta (sideslip angle)
+# are plotted.
+#
+# An equivalent JSBSim XML script version can be found in scripts\rudder_kick.xml
+#
+# Copyright (c) 2023 Sean McLeod
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 3 of the License, or (at your option) any later
+# version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, see <http://www.gnu.org/licenses/>
+#
+
+import jsbsim
+import matplotlib.pyplot as plt
+import math
+
+# Global variables that must be modified to match your particular need
+# The aircraft name
+# Note - It should match the exact spelling of the model file
+AIRCRAFT_NAME="737"
+# Path to JSBSim files, location of the folders "aircraft", "engines" and "systems"
+PATH_TO_JSBSIM_FILES="../.."
+
+# Avoid flooding the console with log messages
+jsbsim.FGJSBBase().debug_lvl = 0
+
+fdm = jsbsim.FGFDMExec(PATH_TO_JSBSIM_FILES)
+
+# Load the aircraft model
+fdm.load_model(AIRCRAFT_NAME)
+
+# Set engines running
+fdm['propulsion/engine[0]/set-running'] = 1
+fdm['propulsion/engine[1]/set-running'] = 1
+
+# Set alpha range for trim solutions
+fdm['aero/alpha-max-rad'] = math.radians(12)
+fdm['aero/alpha-min-rad'] = math.radians(-4.0)
+
+dt = fdm.get_delta_t()
+
+# Max control deflection
+aileronMax = 1
+rudderMax = 0.92
+
+# Number of seconds for control surface to reach max deflection
+risetime = 3
+
+# Per timestep increment for control surfaces
+diAileron = aileronMax / (risetime/dt)
+diRudder = rudderMax / (risetime/dt)
+
+# Recorded data
+times = []
+
+betas = []
+bankAngle = []
+ailerons = []
+rudder = []
+
+# Initial conditions
+fdm['ic/h-sl-ft'] = 1000
+fdm['ic/vc-kts'] = 200
+fdm['ic/gamma-deg'] = 0
+fdm['ic/beta-deg'] = 0
+
+# Initialize the aircraft with initial conditions
+fdm.run_ic() 
+
+# Trim
+try:
+    fdm['simulation/do_simple_trim'] = 1
+
+except RuntimeError as e:
+    # The trim cannot succeed. Just make sure that the raised exception
+    # is due to the trim failure otherwise rethrow.
+    if e.args[0] != 'Trim Failed':
+        raise
```

Comment:
You could use the `TrimFailureError` exception to catach trim failure as that would be more explicit than the legacy code that was used before the `TrimFailureError` exception was introduced by the commit a5484eeea23cb15492b9d73f7ef1e833e025a99c
```suggestion
except jsbsim.TrimFailureError:
    pass  # Ignore trim failures
```

Reply:
Have done. As a separate pull request I'll also update the existing 2 python examples to make use of this trim specific exception.